worlds/saving_princess/Regions.py

```diff
@@ -0,0 +1,126 @@
+from typing import List, Dict
+
+from BaseClasses import MultiWorld, Region, Entrance
+
+from . import Locations
+from .Constants import *
+
+
+class RegionData:
+    location_names: List[str]
+
+    def __init__(self, location_names: List[str]):
+        self.location_names = location_names
+
+
+region_dict: Dict[str, RegionData] = {
+    REGION_MENU: RegionData([]),
+    REGION_CAVE: RegionData(
+        [
+            LOCATION_CAVE_AMMO,
```

Comment:
Is there a specific reason this is a `Dict[str, RegionData]` rather than a `Dict[str, List[str]]`? The surrounding class feels unnecessary, but maybe I'm just missing something here.

Reply:
When I first made the class there was more to it, but you are right that as it currently stands there's no reason for it to exist. I'll do away with it.